src/api/ipc.js

```diff
@@ -42,9 +42,10 @@ class IpcTask {
     }
     this.isProcessing = true;
     this.currentTaskId = this.tasks[0];
-    this.currentTaskInfo = this.tasksInfo[this.currentTaskId].info;
-    this.currentTaskCb = this.tasksInfo[this.currentTaskId].cb;
-    ipcEvent.sender.send('webClientAuthReq', this.currentTaskInfo);
+    const currentTask = currentTask;
```

Comment:
This line must be `const currentTask = this.tasksInfo[this.currentTaskId];`

Reply:
You are right! thanks, I'm pushing the fix for that now.